setup.py

```diff
@@ -136,19 +147,32 @@ def run_tests(self):
     description="Simulates energy using the tranformer AI architecture",
     long_description_content_type="text/markdown",
     url="https://github.com/refereelabs/betafold",
-    cmdclass={"build_ext": CMakeBuild, "clean": CustomCleanCommand, "test": CustomTestCommand},
+    cmdclass={
+        "build_ext": CMakeBuild,
+        "clean": CustomCleanCommand,
+        "test": CustomTestCommand},
     extras_require={
-        "dev": ["mypy==1.15.0", "pytest==8.3.4", "pytest-cov==6.0.0", "coverage==7.6.12", "pre-commit==4.1.0",
-                "clang-format==19.1.7"]},
+        "dev": [
+            "mypy==1.15.0",
+            "pytest==8.3.4",
+            "pytest-cov==6.0.0",
+            "coverage==7.6.12",
+            "pre-commit==4.1.0",
+            "clang-format==19.1.7",
+            "pycodestyle==2.12.1",
+            "autopep8==2.3.2"]},
```

Comment:
The only changes to this file besides the automatically applied pep8 changes are here, I installed `pycodestyle`, which is required for autopep8; and i added `autopep8` as a dependency.